corehq/apps/users/models.py

```diff
@@ -2050,6 +2055,12 @@ def __repr__(self):
             self=self
         ))
 
+    @skippable_quickcache(['self._id'], lambda _: settings.UNIT_TESTING)
+    def get_usercase_id(self):
+        from corehq.apps.hqcase.utils import get_case_by_domain_hq_user_id
+        usercase = get_case_by_domain_hq_user_id(self.domain, self._id, USERCASE_TYPE)
```

Comment:
You make a good point, @snopoke. I looked for a query to fetch just a case ID, and all I found was `get_case_ids_in_domain_by_owner` which also seems like overkill -- it would be nice to be able to specify a case type. Would it be better to use `get_case_ids_in_domain_by_owner`, or would we want to write something new that accepts a case type too?